### ISSUE TITLE
TargetRadius 0 on Android doesnt overlay nested components

### ISSUE THREAD

**Issue by danieljgp2** [prscX/react-native-app-tour#78]
@prscX on iOS when I set TargetRadius on 0 and I have a nested component, the nested componente showns overlay on the outercircle, this doesnt happens on Android.

On Android the nested component is under the outher circle and if I set targetTransparent it Just overlay the content inside the target. 

This is how it looks on android with targetRadius on 0
<img width="318" alt="Captura de Pantalla 2019-08-30 a la(s) 14 51 40" src="https://user-images.githubusercontent.com/24395686/64044828-c75d5a80-cb35-11e9-9b24-71b5f0412d25.png">

This is how it looks on iOS with targetRadius on 0
<img width="330" alt="Captura de Pantalla 2019-08-30 a la(s) 14 50 39" src="https://user-images.githubusercontent.com/24395686/64044831-cc220e80-cb35-11e9-9eac-cc4af4811767.png">

**Comment by witalobenicio** [comment]
@danieljgp2 i think you can just use it. It's open.

**Comment by danieljgp2** [comment]
@witalobenicio it work partially, still not working as it should. Any idea about how to solve it?

**Comment by witalobenicio** [comment]
You are already using my PR?

**Comment by danieljgp2** [comment]
@witalobenicio yes, the problem seems to be related with the change of forView for forBounds, using your pr the component inside of the touchableOpacity with a TargetRadius 0 overlays the outercircle but it freeze the javascript after a while

**Comment by witalobenicio** [comment]
Yeah... I use forView and seems to work just fine in my app.
I don't use forBounds.

**Comment by danieljgp2** [comment]
Using for view doesnt break the title and the description of the tour?

**Comment by witalobenicio** [comment]
Nope. It work just fine.
I don't have the code here to show to you cause I'm on vacations...but as far as I know...it's all good.
You just have to be careful of where you are using. When you use in deep views it doesn't work so nice.
And remember to set collapsable to false.

**Comment by witalobenicio** [comment]
If you have a <Option> component that you want to highlight...maybe the highlight may have to go to the view inside the component.
Got it?

**Comment by danieljgp2** [comment]
Is there a way to use your fork as a dependency? It is the first time i have this kind of issue, i Just took your fork and insert it on node module react native app tour 😁

**Comment by witalobenicio** [comment]
Yes.
When you do a fork, you create a new repo in your own account.
So, for you to use my fork, you just have to add my repo url as a dependency.

react-native-app-tour: <my-fork-git-url>

And you can add with yarn too:

yarn add react-native-app-tour@<my-fork-git-url>

**Comment by danieljgp2** [comment]
Thanks for answering wita, still having the problem on my project, even with your fork.

**Comment by witalobenicio** [comment]
Maybe on Sunday I can help you. I'll be back from vacation.

**Comment by danieljgp2** [comment]
@witalobenicio how can i contact with you?

**Comment by witalobenicio** [comment]
@danieljgp2 lets talk by direct message on twitter. It's the same '@' of GitHub.

**Comment by danieljgp2** [comment]
@witalobenicio Already send you a Twit. Check it out

**Comment by danieljgp2** [comment]
@witalobenicio  You set collapsable false to the view inside the TouchableOpacity that is setted as the appviewtarget or to the appviewtarget itself ?

**Comment by danieljgp2** [comment]
@witalobenicio Im using your fork and it seems to work fine, but when i change from a tab view to another one, opening the tour again freezes the app, it is like the tour is called but not displayed, this doesnt happen when i use forBounds, but with forBounds i have the problem that i just wrote above.

**Comment by witalobenicio** [comment]
@danieljgp2 if you try the same with the default project it works?
Maybe @prscX could help better on this one.

**Comment by danieljgp2** [comment]
@witalobenicio it is working for now, i did a fork and change the forBounds for the forViews, the problem is totally related with this change.
